README.md

```diff
@@ -621,13 +621,38 @@ npm install prosemirror-utils
    ```
 
 
- * **`cloneRowAt`**`(cloneRowIndex: number) → fn(tr: Transaction) → Transaction`\
-   Returns a new transaction that adds a new row after `cloneRowIndex`, cloning the row attributes at `cloneRowIndex`.
+ * **`copyRow`**`(insertNewRowIndex: number, rowToBeClonedIndex: number, options: ?CopyRowOptions) → fn(tr: Transaction) → Transaction`\
+   Returns a new transaction that adds a new row at index `insertNewRowIndex`.
+   Copying the cells from any row using `rowToBeClonedIndex`,
+   you can configure how that copy should working by the options:
+
+   ```javascript
+   options = {
+     /**
+      * if you want to keep the colspan from the cloned row,
+      * by default, it will split the cells.
+      *
+      */
+     keepColspan: false,
+     /**
+      * To keep the deprecated behavior from `cloneRowAt`
+      * that clones the row and increases the rowspan from the previous rows.
+      */
+     expandRowspanFromClonedRow: false
+     /**
+      * You can control what you want to do with the previous cells,
+      * for example, copy the background color when the last cell has a determinate value,
+      * or keep the content for any reason.
+      *
+      * It is not allowed to change the `colspan` or `rowspan` attributes.
+      * Those values will always be changed to the original value.
+      * You can use other flags to control it.
+      */
+     getNewCell: (copyFromPreviousCell) => cell,
```

Comment:
What are the use cases? Do we really need it for anything?

Reply:
I added a use case on the description, but we can copy no even the attributes but the content to:

![Screen Recording 2019-08-05 at 11 47 am](https://user-images.githubusercontent.com/3452187/62433216-dc351100-b776-11e9-8918-113c01263ec7.gif)

but is not the `prosemirror-utils` who is going to decide that, to solve it you give the user the freedom.

Reply:
why would you want to copy the content?

Reply:
that's up to the user, right now this is possible.